pdfmaker/kassia.py

```diff
@@ -77,18 +79,24 @@ def createPDF(self):
                 c.setFillColor(title_attrib['color'])
                 
                 vert_pos -= title_attrib['font_size'] + 10
-                c.setFont('EZOmega',title_attrib['font_size'])
+                c.setFont(title_attrib['font'],title_attrib['font_size'])
                 c.drawCentredString(self.paperSize[0]/2,vert_pos,title_text)
                 vert_pos -= 36
             c.setFillColor(colors.black)
 
-            neumesText = " ".join(troparion.find('neumes').text.strip().split())
-            lyricsText = " ".join(troparion.find('lyrics').text.strip().split())
+            # Get attributes for neumes
+            neume_elem = troparion.find('neumes')
+            if (neume_elem is not None):
+                neumesText = " ".join(neume_elem.text.strip().split())
+                neume_attrib = neume_elem.attrib
+                neume_attrib = self.fill_title_dict(neume_attrib)
```

Comment:
Doesn't have to be in this PR, but we should toss up a warning if the element is non-existent. I could see someone not wanting lyrics (blank templates for prosomia), but neumes seem like a "must have" to me. Thoughts?